web/app/components/new/doc-form.ts

```diff
@@ -148,10 +135,17 @@ export default class NewDocFormComponent extends Component<NewDocFormComponentSi
       this.productArea = formObject["productArea"] as string;
     }
 
-    if (this.summary.length > 200) {
-      this.summaryIsLong = true;
-    } else {
-      this.summaryIsLong = false;
```

Comment:
I removed this because it no longer seemed necessary.

We initially implemented it to reduce the likelihood of [unclean string] errors (i.e., strings with special characters). Yes, we also wanted succinct summaries, but that wasn't the primary driver, nor was it the problem.

Any thoughts?

Reply:
Hmm, I think this was because we saw issues with the doc header rewrites if there was a very long summary - I haven't tested that since we added this restriction in the frontend though. Can we add this back for now? The backend should be enforcing some length maximum but isn't currently (then we can maybe remove this?).

Reply:
Added it back, but to be clear, this isn't a restriction,but a suggestion (though it'd take a real REBEL to disregard the warning text).